src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
@timbussmann @andreasohlund The outbox replaces a behavior and then calls that replaced behavior. That required to register this replaced behavior in the outbox feature.

This is a breaking change if the customer replaces behaviors as that replaced behavior is now not by default configured.

We can also add a feature called `BackwardCompatibleBehaviorConfiguration`. This can then be enabled by default but be disabled by the customer to make sure we do not have incomplete registrations? That would be a bump in the minor version I guess based on semver as we are extending the API.

So that would then become NServiceBus 5.3.0 if we want to release this, be fully backwards compatible but have the ability to disabled it.